### PR TITLE
Link to support page

### PR DESCRIPTION
**Pull request by everett-toews**
Rather than have a paragraph of text in the left-hand nav we should keep it simple and just link to our support page. This is also consistent with the nav in the upper menu Help > Developer-to-Developer Support

### CODE REVIEW

**Comment by rgbkrk** [comment]
This seems like something I'd want to run by @ChristieQ in a flight.

**Comment by ChristieQ** [comment]
Thanks,@rgbkrk! According to heat map data, the only items in the left nav that get any clicks at all are the page navigation (i.e., languages in the SDKs section of the site and product names in the API docs section). Based on that data, we expect that changes to this section of the page will have little to no impact. Our plan is to move all of this content in the upcoming redesign, based on the results of user testing.

TL;DR: We can make the change, but chances are good that it will have little impact on CX.

**Comment by everett-toews** [comment]
I'm really just looking for consistency in how we direct users to support.
